client/components/404/404.js

```diff
@@ -4,39 +4,36 @@ import "./index.css";
 import PropTypes from "prop-types";
 import React from "react";
 import {Link} from "react-router-dom";
-
-import getText from "../../utils/get-text";
+import {t} from "ttag";
 
 export default class DoesNotExist extends React.Component {
   componentDidMount() {
-    const {orgName, setTitle, page, language} = this.props;
+    const {orgName, setTitle, page} = this.props;
     if (page !== undefined || orgName !== undefined)
-      setTitle(page, language, orgName);
+      setTitle(t`404_PAGE_TITLE`, orgName);
   }
 
   render() {
-    const {orgSlug, language, page} = this.props;
+    const {orgSlug, page} = this.props;
     return (
       <div className="container content" id="not-foud-404">
         <div className="inner">
           <div className="main-column">
             <div className="row owisp-404-row-1">
-              {page && page.heading ? getText(page.heading, language) : "Oops!"}
+              {page && page.heading ? t`404_HEADING` : "Oops!"}
```

Comment:
I think we can remove these default values at the end right? Let's try to shorten the code a bit where the old code is not necessary anymore, keep in mind we have to start thinking in reducing the size of the build.

Reply:
I think No, because these default values are used on those URL where organization-slug is missing like `http://0.0.0.0:8080/error/` as setOrganization is not called. Also 404 page of organization and without organization is different.